camera-viewfinder-compose/src/main/java/com/google/jetpackcamera/viewfinder/CameraPreview.kt

```diff
@@ -72,49 +84,108 @@ fun PreviewSurface(
     modifier: Modifier,
```

Comment:
Actually, shall we make PreviewSurface (and TransformedSurface) private if we don't expect them to be used externally?